doc/cookbook/source/examples/classifier/kernel_svm.rst

```diff
@@ -0,0 +1,57 @@
+=============================
+Kernel Support Vector Machine
+=============================
+
+Kernel Support Vector Machine is a binary classifier which finds a data-separating hyperplane in a Hilbert space induced by a positive definite kernel. The hyperplane is chosen to maximize the margins between the two classes. The loss function that needs to be minimized is:
+
+.. math::
+
+    \max_{\bf \alpha} \sum_{i=0}^{N-1} \alpha_i - \sum_{i=0}^{N-1}\sum_{j=0}^{N-1} \alpha_i y_i \alpha_j y_j  k({\bf x}_i, {\bf x}_j)
+
+subject to:
+
+.. math::
+
+    0 \leq \alpha_i \leq C, \sum_{i=0}^{N-1} \alpha_i y_i = 0
+
+where :math:`N` is the number of training samples, :math:`{\bf x}_i` are training samples, :math:`k` is a kernel, :math:`\alpha_i` are the weights, :math:`y_i` is the corresponding label where :math:`y_i \in \{-1,+1\}` and :math:`C` is a pre-specified regularization parameter.
+
+Implementation of Kernel SVM in Shogun uses LibSVM :cite:`chang2011libsvm`.
```

Comment:
Say "This example uses LibSVM as backend, Shogun has many more SVM implementation, see CSVM"